src/python/zquantum/core/wip/circuit/conversions/cirq_conversions_test.py

```diff
@@ -0,0 +1,156 @@
+import cirq
```

Comment:
Just a thought here, perhaps no action necessary, but should there be an inherited set of tests to ensure all translation packages work in the same way? Perhaps this is relevant to @mstechly 's tests for gates

Reply:
At the moment, we're not 100% sure our current abstractions are well suited for wrapping all the frameworks. It'd be less risky to consider such tests when we finish conversion routines, let's come back to this idea at that time. Good suggestion, though!